pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -3087,6 +3087,8 @@ fn multiple_custodian_settlement() {
 
 // ----------------------------------------- Confidential transfer tests -----------------------------------
 
+/// This function creates a mercat account and returns its secret part (to be stored in the wallet) and
```

Comment:
```suggestion
/// Creates a mercat account and returns its secret part (to be stored in the wallet) and
```
(and elsewhere)

Adding "This function" before functions doesn't add anything; its better to be more direct and concise in docs.